Pages/product-listing.html

```diff
@@ -0,0 +1,305 @@
+                            </div>
+                        </div>
+                    </div> 
+                </div>
+                <div class="card margin--medium">
+                    <div class="card__image--container">
+                        <img class="card__image " src="../Images/book1.jpg" />
+                    </div>
+                    <div class="card__body padding--medium">
+                        <div class="card__heading">
+                            <div class="container__flex--spacebetween">
+                                <h2 class="card__title text--large">Product</h2>
+                                <i class="far fa-heart text--large"></i>
+                            </div>
+                        </div> 
+                        <div class="card__description margin-bottom--small">
+                            <p class="clr--secondary  text--bold">Rs. 599 <span class="text--line-through"> 999 </span></p>
+                        </div>
+                        <div class="card__actions">
+                            <div class="card__actions--buttons">
+                                <button class="btn btn--primary border--grey">View Product</button> 
+                                <button class="btn btn--secondary btn--icon"><i class="fas fa-shopping-cart"></i>Add to Cart</button>                    
+                            </div>
+                        </div>
+                    </div> 
+                </div>
+                <div class="card margin--medium">
+                    <div class="card__image--container">
+                        <img class="card__image " src="../Images/book1.jpg" />
+                    </div>
+                    <div class="card__body padding--medium">
+                        <div class="card__heading">
+                            <div class="container__flex--spacebetween">
+                                <h2 class="card__title text--large">Product</h2>
+                                <i class="far fa-heart text--large"></i>
+                            </div>
+                        </div> 
+                        <div class="card__description margin-bottom--small">
+                            <p class="clr--secondary  text--bold">Rs. 599 <span class="text--line-through"> 999 </span></p>
+                        </div>
+                        <div class="card__actions">
+                            <div class="card__actions--buttons">
+                                <button class="btn btn--primary border--grey">View Product</button> 
+                                <button class="btn btn--secondary btn--icon"><i class="fas fa-shopping-cart"></i>Add to Cart</button>                    
+                            </div>
+                        </div>
+                    </div> 
+                </div>
+            </div>
+
+        </div>
+
+        <div class="footer__container padding--medium">
```

Comment:
Everything seems good to me, I have noticed one thing that you haven't used semantic tags in html structure. For e.g.- for footer, we can use <footer> tag and for navigation <nav> tag.

Reply:
thanks, yes semantic html tags are good practise. Will refactor the code to use sections